src/openrct2/ride/Ride.cpp

```diff
@@ -8131,17 +8788,14 @@ uint8 ride_entry_get_first_non_null_ride_type(rct_ride_entry * rideEntry)
             return rideEntry->ride_type[i];
         }
     }
-   return RIDE_TYPE_NULL;
+    return RIDE_TYPE_NULL;
 }
 
 bool ride_type_supports_boosters(uint8 rideType)
 {
-    if (rideType == RIDE_TYPE_LOOPING_ROLLER_COASTER ||
-        rideType == RIDE_TYPE_CORKSCREW_ROLLER_COASTER ||
-        rideType == RIDE_TYPE_TWISTER_ROLLER_COASTER ||
-        rideType == RIDE_TYPE_VERTICAL_DROP_ROLLER_COASTER ||
-        rideType == RIDE_TYPE_GIGA_COASTER ||
-        rideType == RIDE_TYPE_JUNIOR_ROLLER_COASTER)
+    if (rideType == RIDE_TYPE_LOOPING_ROLLER_COASTER || rideType == RIDE_TYPE_CORKSCREW_ROLLER_COASTER ||
```

Comment:
This clang autoformatting makes stuff harder to read. Can it not be configured to make the if statement vertical if it's too long?